docs/static/spec/openapi/logstash-api.yaml

```
@@ -170,60 +323,210 @@ paths:
-                            peak_max_in_bytes: 286326784
-                            max_in_bytes: 286326784
-                            committed_in_bytes: 71630848
-                      gc:
-                        collectors:
-                          old:
-                            collection_time_in_millis: 607
-                            collection_count: 12
-                          young:
-                            collection_time_in_millis: 4904
-                            collection_count: 1033
-                      uptime_in_millis: 1809643
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: '#/components/schemas/JVMStats'
+                  - $ref: '#/components/schemas/ProcessStats'
+                  - $ref: '#/components/schemas/EventsStats'
+                  - $ref: '#/components/schemas/FlowStats'
+                  - $ref: "#/components/schemas/PipelineStats"
+                  - $ref: '#/components/schemas/QueueStats'
+                  - $ref: '#/components/schemas/ReloadStats'
+
+  /_node/stats/events:
+    get:
+      summary: Get events statistics
+      description: Return statistics related to event processing.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing events statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/EventsStats"
+
+  /_node/stats/flow:
+    get:
+      summary: Get flow statistics
+      description: Return throughput and backpressure details.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing flow statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/FlowStats"
+
+  /_node/stats/jvm:
+    get:
+      summary: Get flow statistics
+      description: Return throughput and backpressure details.
+      tags:
+        - node stats
+      responses:
+        "200":
+          description: A JSON object containing jvm statistics.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Common"
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - $ref: "#/components/schemas/JVMStats"
+
+  /_node/stats/os:
+    get:
+      summary: Get Cgroup statistics
+      description: Return a more accurate view of CPU statistics from Cgroup if available.
```

Is "Cgroup" the same as control group? If yes, IMO it's worth spelling out at least once.
```suggestion
      description: Get a more accurate view of CPU statistics from control groups (Cgroup) if available.
```

---

Yes